components/tabBarComponents/EventTabBar.tsx

```diff
@@ -16,8 +16,8 @@ export const EventTabBar = () => {
         <TouchableOpacity style={styles.icon}>
           <Ionicons name="chatbox" size={iconSize} color={"grey"} />
         </TouchableOpacity>
-        <TouchableOpacity style={styles.icon}>
-          <Ionicons name="add-circle" size={60} color={darkIconColor} />
+        <TouchableOpacity style={styles.plusIcon}>
+          <Ionicons name="add-outline" color="white" size={30} />
```

Comment:
We have a `TouchableIonicon` component for this, which also scales based on the user's font settings.